src/cavendish_particle_tracks/_widget.py

```diff
@@ -296,6 +304,28 @@ def _on_click_stereoshift(self) -> StereoshiftDialog:
         dlg.move(point)
         return dlg
 
+    def _on_click_load_data(self) -> None:
+        """When the 'Load data' button is clicked, a dialog opens to select the folder where the data is.
+        The data in the folder is loaded as a stack of images, and the stack is named according to the option selected.
+        """
+
+        if self.load.currentIndex() < 1:
+            return
+
+        folder_name = QFileDialog.getExistingDirectory(
+            self, "Open Folder", "./"
+        )
+        stack = imread(folder_name + "/*")
+        self.viewer.add_image(stack, name=self.load.currentText())
+
+        # stack_view1 = imread("./tests/data/View1*.tiff")
+        # stack_view2 = imread("./tests/data/View2*.tiff")
+        # self.imview1 = self.viewer.add_image(stack_view1, name="View1")
+        # self.imview2 = self.viewer.add_image(stack_view2, name="View2")
+        # TODO: investigate the multiscale otption.
+
+        self.load.setCurrentIndex(0)
```

Comment:
I get a stacktrace when I try opening your test data...
```
---------------------------------------------------------------------------
ValueError                                Traceback (most recent call last)
~/personal-projects/cavendish-particle-tracks/src/cavendish_particle_tracks/_widget.py in ?(self=<cavendish_particle_tracks._widget.ParticleTracksWidget object>)
    314
    315         folder_name = QFileDialog.getExistingDirectory(
    316             self, "Open Folder", "./"
    317         )
--> 318         stack = imread(folder_name + "/*")
    319         self.viewer.add_image(stack, name=self.load.currentText())
    320
    321         # stack_view1 = imread("./tests/data/View1*.tiff")
```

Presumably something is missing from `pyproject.toml`?